### DISCUSSION TITLE
Bug: unable to save number attributes with fractional values

### DISCUSSION THREAD

**Discussion by mewa** [swellstores/community#283] [Help]
Hi,

for a store which has PL locale set (which I think is the root cause of the problem here) we're unable to save attribute values that contain fractional parts.

You see, PL locale uses commas rather than dots for separating the fractional part of the number. The admin panel is in English and the input fields only accept dots as the separator. Using dots instead of commas wouldn't be a problem if it was consistent.

But it's not.

When entering `29.5` as a value of a Number attribute and saving it, after refreshing the product page the actual value that has been saved is `295`.

This only happens in the PL store, in the store using EN locale it works just fine which is why I think it's tied to the store's locale.

**Comment by SusannaSwell** [comment]
@mewa Thanks for bringing it to us. I confirm that it is a bug and let me create a bug ticket internally for faster resolution.
